bin/Cargo.toml

```diff
@@ -93,8 +93,8 @@ tls = ["dns-over-openssl"]
 # WARNING: there is a bug in the mutual tls auth code at the moment see issue #100
 # mtls = ["hickory-client/mtls"]
 
-webpki-roots = ["hickory-client/webpki-roots"]
-native-certs = ["hickory-client/native-certs"]
+webpki-roots = ["hickory-client/webpki-roots", "hickory-server/webpki-roots"]
```

Comment:
And then these should also be weak?

Reply:
Both `hickory-client` and `hickory-server` are non-optional here